src/screens/AboutPage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Navbar from '../components/Navbar/Navbar';
+
+function AboutPage() {
```

Comment:
Since you're using typescript remember to have the return type for your component.
Example:

```
function AboutPage() : JSX.Element {
  ...
}
```